featuretools/tests/entityset_tests/test_es.py

```diff
@@ -2016,3 +2017,19 @@ def test_dataframe_type_empty_es():
 
 def test_dataframe_type_pandas_es(pd_es):
     assert pd_es.dataframe_type == Library.PANDAS.value
+
+
+def test_pd_es_pickling(pd_es):
+    pkl = pickle.dumps(pd_es)
+    unpickled = pickle.loads(pkl)
+
+    assert pd_es.__eq__(unpickled, deep=True)
+    assert not hasattr(unpickled, 'schemas')
```

Comment:
This instance of the magic schemas key should probably also be using a module-level constant that is imported from the `entityset` module.